src/sdk/data/data.module.ts

```diff
@@ -171,6 +171,38 @@ export class DataModule {
     }
   }
 
+  async getExchangeOffers(
+    fromTokenAddress: string,
+    toTokenAddress: string,
+    fromAmount: BigNumber,
+    fromChainId: number,
+    fromAddress: string,
+    toAddress?: string,
+    showZeroUsd?: boolean
+  ): Promise<ExchangeOffer[]> {
+    const account = fromAddress;
+
+    try {
+      const queryParams = {
+        'api-key': this.currentApi,
```

Comment:
We seem to be using `api-key`, `apiKey` and `api_key` at several places within Etherspot hosted services. Can we stick to one name please?

Reply:
The `api-key` parameter in the DataUtils module is configured by **TYK** itself, not by us.